evm/ts-scripts/.env.sample

```
@@ -0,0 +1,3 @@
+WALLET_KEY=
+LEDGER_BIP32_PATH=
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The LEDGER_BIP32_PATH key should go before the WALLET_KEY key

(UnorderedKey)